articles/inter-vlan-routing/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/inter-vlan-routing/
+title: Inter-VLAN routing
+description: This article is viewed at helping one to gain the understanding of inter-vlan routing and how to configure it using a packet tracer, add fields `dateCreated` and `lastModified` in the `Todo` data class.
+author: atonya-dennis
+date: 2021-03-16T00:00:00-10:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/inter-vlan-routing/hero.jpg
+    alt: Network Connection example image
+---
+As a network administrator, one of the tasks is to segment a single switched Local Area Network (LAN) logically without having to run new cables and changing the existing network. The concept of "VLAN" and its configurations will help one handle the task at hand with ease.
+<!--more-->
+In this article, we will learn about what inter-VLAN routing is, various methods for VLAN routing, and also build a network configuration to understand how it works.
+
+# Table of contents
+- [Prerequisite](#prerequisite)
+- [What is inter-VLAN routing?](#what-is-inter-vlan-routing)
+- [Methods of inter-VLAN routing](#inter-vlan-routing-methods)
+- [Inter-VLAN network configurations](#network-configurations-for-inter-vlan-communication-using-router-on-stick-method)
+- [Conclusion](#conclusion)
+
+### Prerequisite
+As a prerequisite, the reader must have a good understanding of basic networking concepts, and how to work with Cisco packet tracer.
+
+To download Cisco packet tracer visit [this link](https://www.computernetworkingnotes.com/ccna-study-guide/download-packet-tracer-for-windows-and-linux.html) to install the latest version installed on your computer.
+
+### What is Inter-VLAN routing?
+Virtual Local Area Networks (VLANs) are network segments on a switched LAN that groups different hosts together logically, regardless of their physical locations on the network.
+
+VLANs make it easier to segment a workstation into different isolated LANs. They improve the performance of the network and are flexible, since they are logically connected.
+
+VLANs act as separate subnets on the network. To move packets from one VLAN to another and enable communications between hosts, we have to configure the network to enable inter-VLAN routing.
+
+### Inter-VLAN routing methods
+#### Legacy inter-VLAN routing
+In this method, multiple router interfaces are used, each connecting to a switch port in different VLANs. These interfaces are served as default gateways. Sometimes, it may lead to network issues, since expanding the network requires additional cabling.
+
```

Comment:
How adding additional cables would lead to network issues? Could you explain briefly?